src/components/Characters/Character.js

```diff
@@ -2,27 +2,53 @@ import React, {useState} from 'react'
 import './Character.css'
 
 const Character = ({props, index}) => {
-    const {image,name,status,species,gender} = props
+    const {image,name,status,species,gender, episode} = props
 
     const [TouchImage, setTouchImage] = useState(false)
 
     const HandlerTouch = (event) => {
```

Comment:
why it's capitalized? should be lowerCamelCase

Reply:
sure